### ISSUE TITLE
Remove "-package" suffix from Crossplane packages name

### ISSUE THREAD

**Issue by web7evgheni** [kndpio/kndp#143]
## Issue Summary:
Every Crossplane package is OCI image where we could store any data, and use it in same time for Crossplane for Pod or for Helm package. In this case i see no sense to use -package prefix, because scope of OCI image in this case comes from usage context.
## Todo:
- [ ] Make Crossplane packages to be available under their pure names. 

## Acceptance Criteria:
- User shall be easy use Crossplane package without add suffix.

**Comment by web7evgheni** [comment]
@ca7alindev Is renaming of package name solved problem?

**Comment by ca7alindev** [comment]
Yes, name should not use "-package" suffix anymore, i solved it in last pr
